src/components/Footer.js

```diff
@@ -1,8 +1,127 @@
-import React from "react";
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedin, faGithub } from '@fortawesome/free-brands-svg-icons';
 
 const Footer = () => {
+  const currentYear = () => {
+    const today = new Date();
+    const year = today.getFullYear();
+    return year;
+  };
+
   return (
-    <h1> Hello This is Footer component </h1>
+    <footer className="bg-primary text-white pt-3">
+      <div className="d-flex justify-content-around">
+        <section>
+          <h6 className="text-uppercase">Social</h6>
+          <ul className="list-unstyled mb-1">
+            <li>
+              <p className="mb-0">
+                <a
+                  className="github social text-white"
+                  href="https://github.com/CodeYourFuture/project-quiz/"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  <FontAwesomeIcon
+                    aria-hidden="true"
+                    aria-label="Github icon"
```

Comment:
We don't need `aria-label` anymore. `aria-label` creates a label for screen readers, but since we are hiding this icon from screen readers with `aria-hidden="true"` then we don't need the `aria-label` anymore. You can remove this in the rest of the icons in this file